src/main/java/core/mate/academy/model/ExcavatorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.model;
+
+import java.util.Random;
+
+public class ExcavatorSupplier {

```

Comment:
This kind of class must implement the interface MachineProducer, but not machine models